src/people.js

```diff
@@ -2,8 +2,22 @@
 
 /* eslint-disable */
 const people = [
-  { 'name': 'Carolus Haverbeke', 'sex': 'm', 'born': 1832, 'died': 1905, 'father': 'Carel Haverbeke', 'mother': 'Maria van Brussel' },
-  { 'name': 'Emma de Milliano', 'sex': 'f', 'born': 1876, 'died': 1956, 'father': 'Petrus de Milliano', 'mother': 'Sophia van Damme' },
+  {
+    'name': 'Carolus Haverbeke',
+    'sex': 'm',
+    'born': 1832,
+    'died': 1905,
+    'father': 'Carel Haverbeke',
+    'mother': 'Maria van Brussel'
+  },
```

Comment:
Add a trailing comma after the last item in the object to follow the `comma-dangle` rule.